### DISCUSSION TITLE
Postal shows Hard Fail / Mautic shows email read -> url hit

### DISCUSSION THREAD

**Discussion by revcycpro** [postalserver/postal#2070] [Help with using Postal]
Hi, 
Form past few days I see that Postal marks the email as Hard Fail, but my email marketing tool "Mautic" shows the email has been read and the link have been visited, 

I have tried this case, many time on the same email (including several other marked as hardfail) and the results are same. I also verified with two of my customers, they said they are receiving the email, but on postal it is marked as hard fail. This is also increasing the bounce %.

I am confused. What to do? 
Kindly help.

**Comment by willpower232** [comment]
Something must be getting confused. Does Postal tell you what the reason is for the fail?

**Reply by revcycpro** [reply]
The reason is this (But this bad reputation was built because postal started marking this as hard fail) - "
550-5.7.1 [2605:a140:2076:4407::1 19] Our system has detected that this 550-5.7.1 message is likely suspicious due to the very low reputation of the 550-5.7.1 sending domain. To best protect our users from spam, the message has 550-5.7.1 been blocked.

**Reply by revcycpro** [reply]
My emails used to deliver on gmail server as well, but from last week hell broke loose. I have all my emails verified from vetted sources and the content is not at all spammy. Any way to get rid of it. Is it because of no ssl cert on server?

**Reply by revcycpro** [reply]
The reason a two-weeks ago (I corrected this 2-3 days later)- 550-5.7.25 [2605:a140:2076:4407::1] The IP address sending this message does not 550-5.7.25 have a PTR record setup, or the corresponding forward DNS entry does 550-5.7.25 not point to the sending IP. As a policy, Gmail does not accept 550-5.7.25 messages from IPs with missing PTR records. Please visit 550-5.7.25 -

**Reply by willpower232** [reply]
If you're looking at gmail exclusively, its likely that Google is loading all the image assets in the email and clicking all the links to verify the content of the email so that is probably why you're seeing both the fail and the message being accessed.

It might be worth using mxtoolbox.com to verify that there are no easily fixable issues on the server. You could also try https://www.mail-tester.com/ or similar to check your messages are as good as they can be.
